addons/binding/org.openhab.binding.lgwebos/ESH-INF/thing/thing-types.xml

```diff
@@ -17,8 +17,11 @@
 			<channel id="mediaPlayer" typeId="mediaPlayerType" />
 			<channel id="mediaStop" typeId="mediaStopType" />
 			<channel id="appLauncher" typeId="appLauncherChannelType" />
-
 		</channels>
+
+        <representation-property>deviceId</representation-property>
```

Comment:
Please use tabs to indent the XML. 

After running `mvn clean install` on the command line you can easily spot such issues in the static code analysis report (`target/code-analysis/report.html`).